index.ts

```diff
@@ -16,8 +16,16 @@ function pageScript(): void {
 	button.addEventListener('click', _ => {
 		window.close();
 	});
+
+	let timeout: ReturnType<typeof setTimeout>;
 	window.addEventListener('blur', _ => {
-		window.close();
+		timeout = setTimeout(() => {
+			window.close();
+		}, 60_000);
+	});
+
+	window.addEventListener('focus', _ => {
+		clearInterval(timeout);
```

Comment:
Wait a minute before auto-closing on blur. We just don't want the windows to lay around forever.